src/main/java/net/bestemor/villagermarket/menu/Shopfront.java

```diff
@@ -255,6 +255,8 @@ public void onClick(InventoryClickEvent event) {
                         shop.openInventory(player, ShopMenu.EDIT_SHOP);
                     } else if (event.getClick() == ClickType.RIGHT) {
                         event.getView().close();
+                    } else if (ConfigManager.getBoolean("disable_lore_toggle")) {
```

Comment:
This will re-open the customer menu every time you click the empty slot. I think the best solution would be to add `else if (!ConfigManager.getBoolean("disable_lore_toggle"))` at line `253` (The `else` after `if (type == Type.EDITOR)`). This would disable all unintended actions with the empty slot.